convert2rhel/repo.py

```diff
@@ -131,15 +130,22 @@ def write_temporary_repofile(contents):
     :raises exceptions.CriticalError: In case of not being able to write the
         repository contents to a file.
     """
-    repofile_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=TMP_DIR)
+    try:
+        repofile_dir = tempfile.mkdtemp(prefix="downloaded_repofiles.", dir=TMP_DIR)
+    except (OSError, IOError) as err:
+        raise exceptions.CriticalError(
+            id_="CREATE_TMP_DIR_FOR_REPOFILES_FAILED",
+            title="Failed to create a temporary directory",
+            description="Failed to create a temporary directory for storing a repository file under %s.\n"
+            "Reason: %s" % (TMP_DIR, str(err)),
```

Comment:
```suggestion
            diagnosis="Reason: %s" % (TMP_DIR, str(err)),
```
Shouldn't this be diagnosis?

Reply:
Good point. I've read the notification too late. It's merged now but you can raise that under https://issues.redhat.com/browse/RHELC-1598.